services/hierarchy/src/test/kotlin/OrtRunServiceTest.kt

```diff
@@ -178,6 +181,64 @@ class OrtRunServiceTest : WordSpec() {
                 fixtures.ortRunRepository.get(ortRunId) shouldNotBe null
             }
         }
+
+        "deleteAncientRuns" should {
```

Comment:
The function has been renamed to "deleteRunsCreatedBefore"; this should be reflected here as well.